app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int
+                 ):
+        self.count_of_ratings = count_of_ratings
+        self.average_rating = average_rating
+        self.clean_power = clean_power
+        self.distance_from_city_center = distance_from_city_center
+
+    def serve_cars(self, cars):
+        total_cost = []
```

Comment:
You don't really to create a new list to calculate sum of costs. Avoid using extra list, just store total cost as a numeric variable